docs-ref-mapping/reference-unified.yml

```diff
@@ -45,6 +45,11 @@
       landingPageType: Service
       children:
       - 'azure-mgmt-applicationinsights'
+  - name: Anomaly Detector
+    uid: azure.python.sdk.landingPage.services.AnomalyDetector
+    href: ~/docs-ref-services/ai-anomalydetector-readme-pre.md
```

Comment:
```suggestion
    href: ~/docs-ref-services/ai-anomalydetector-readme.md
```

Reply:
The `reference-unified` is the core that get's applied. `-pre` will be added when generating the readme ToC. ([example](https://github.com/MicrosoftDocs/azure-docs-sdk-python/blob/db8adf85ab49e224038de79e10278b8b67b8d45b/docs-ref-mapping/reference-preview.yml#L12) of exactly what I'm talking about.)

Otherwise, this is awesome!